PWGLF/Tasks/Strangeness/derivedlambdakzeroanalysis.cxx

```diff
@@ -459,6 +467,22 @@ struct derivedlambdakzeroanalysis {
     if (fabs(v0.negTOFDeltaTK0Pi()) < maxDeltaTimePion)
       bitset(bitMap, selTOFDeltaTNegativePionK0Short);
 
+    // TOF PID
+    // Positive track
+    if (fabs(v0.posTOFDeltaTLaPr()) < maxDeltaTimeProton)
```

Comment:
Hi @romainschotter, is this not equivalent to the preceding block? Or maybe I am missing something?... sorry if so!